src/service/routes/auth.js

```diff
@@ -3,6 +3,13 @@ const router = new express.Router();
 const passport = require('../passport').getPassport();
 const db = require('../../db');
 const passportType = passport.type;
+const { GIT_PROXY_UI_HOST: uiHost = 'http://localhost', NODE_ENV } = process.env;
+
+// TODO: Refactor this through proper .env loading. This handles redirects in dev.
+let uiPort = 3000;
+if (NODE_ENV === 'production') {
+  uiPort = process.env.GIT_PROXY_UI_PORT;
+}
```

Comment:
Directly assign to `uiPort` from process.env with a default in that assignment line instead of wrapping it in a `NODE_ENV` based check.
```suggestion
const { GIT_PROXY_UI_HOST: uiHost = 'http://localhost', GIT_PROXY_UI_PORT: uiPort = 3000 } = process.env;
```

Reply:
Thank you!